frontend/src/scenes/session-recordings/player/PlayerSettings.tsx

```diff
@@ -0,0 +1,108 @@
+
+import { playerSettingsLogic } from './playerSettingsLogic'
+import { PLAYBACK_SPEEDS } from './sessionRecordingPlayerLogic'
+
+export const PlayerSettings = (): JSX.Element => {
+    const { speed, autoplayDirection, skipInactivitySetting, showMouseTail, showSeekbarPageviews } =
+        useValues(playerSettingsLogic)
+    const { setSpeed, setAutoplayDirection, setSkipInactivitySetting, setShowMouseTail, setShowSeekbarPageviews } =
+        useActions(playerSettingsLogic)
+
+    return (
+        <div className="pl-2">
+            <LemonMenu
+                closeOnClickInside={false}
+                items={[
+                    {
+                        custom: true,
+                        label: () => (
+                            <div className="flex flex-row items-center justify-between space-x-2 p-1 pl-1 min-w-64">
+                                <span className="text-black font-medium">Autoplay</span>
+
+                                <LemonSelect
+                                    tooltip={
+                                        <div className="text-center">
+                                            Autoplay next recording
+                                            <br />({!autoplayDirection ? 'off' : autoplayDirection})
+                                        </div>
+                                    }
+                                    value={autoplayDirection}
+                                    aria-label="Autoplay next recording"
+                                    onChange={setAutoplayDirection}
+                                    dropdownPlacement="bottom-end"
+                                    dropdownMatchSelectWidth={false}
+                                    options={[
+                                        { value: null, label: 'Off' },
+                                        { value: 'newer', label: 'Newer recordings' },
+                                        { value: 'older', label: 'Older recordings' },
+                                    ]}
+                                    size="small"
+                                />
+                            </div>
+                        ),
+                    },
+                    {
+                        custom: true,
+                        label: () => (
+                            <LemonSwitch
+                                className="px-2 py-1"
+                                checked={showMouseTail}
+                                onChange={setShowMouseTail}
+                                label="Show mouse tail"
+                                fullWidth
+                            />
+                        ),
+                    },
+                    {
+                        custom: true,
+                        label: () => (
+                            <LemonSwitch
+                                className="px-2 py-1"
+                                checked={showSeekbarPageviews}
```

Comment:
this makes me think "why only pageviews, why not other things instead" but no need to add that complexity now

Reply:
I'd say at least `$screen` events as well.
every time you think of $pageview we should think of $pageview or $screen so it works for Mobile as well.

Reply:
I went with $pageview because they seem to be the ones that occur most frequently and cause the seekbar to get a little messy.

Good point about $screen, it should be included. Is there a generic term we use to describe Pageview and Screen events collectively?

Reply:
Good question, I don't think so, all filters have to be done separately so I don't think the generic term exists, this might be the first feature that would need to address both cases together I guess.
IMO Pageview and Screen should be unified, they are just synonyms but still different events, so depending on the platform you have to use one or the other.
Maybe just calling "Pageview and Screen events" would be the best option now?

Reply:
Combined both in the latest commit